proxy-node-translator/src/main/java/uk/gov/ida/notification/translator/saml/EidasResponseGenerator.java

```diff
@@ -22,12 +22,9 @@
     private final MetatronProxy metatronProxy;
     private final X509CertificateFactory x509CertificateFactory;
 
-    public EidasResponseGenerator(
-            HubResponseTranslator hubResponseTranslator,
-            EidasFailureResponseGenerator failureResponseGenerator,
-            SamlObjectSigner samlObjectSigner,
-            MetatronProxy metatronProxy,
-            X509CertificateFactory x509CertificateFactory) {
+    public EidasResponseGenerator(final HubResponseTranslator hubResponseTranslator,
```

Comment:
All of these should be on separate lines - this seems to be the convention we use when constructors have many arguments. I can see a few different styles in this PR - we should be consistent with style as much as possible